tozny/resource_account.go

```diff
@@ -39,11 +214,39 @@ func resourceAccount() *schema.Resource {
         ForceNew:    true,
         Elem: &schema.Resource{
           Schema: map[string]*schema.Schema{
-            "id": {
-              Description: "The unique user identifier for the account.",
-              Type:        schema.TypeInt,
-              Computed:    false,
+            "company": {
+              Description: "The unique server defined identifier for the account.",
+              Type:        schema.TypeString,
+              Optional:    true,
+            },
+            "plan": {
+              Description: "User defined identifier for the account.",
+              Type:        schema.TypeString,
+              Required:    true,
```

Comment:
I am pretty sure this can be optional. Billing has default values that it uses.

Reply:
updated